src/borg/archiver.py

```diff
@@ -3486,6 +3497,9 @@ def define_borg_mount(parser):
         with the ``--cache-only`` option, or keep the security info with the
         ``--keep-security-info`` option.
 
+        When in doubt, use ``--dry-run --verbose --list`` to see how patterns/PATHS are
+        interpreted. See :ref:`list_item_flags` in ``borg create`` for details.
```

Comment:
when --list is used, does it really also require --verbose?

please check for delete and also for other borg subcommands and do it in a consistent way.

Reply:
also, the item_flags are not relevant for borg delete (ARCHIVE|REPO).

Reply:
No I don't think it requires --verbose, however it is present by default in all subparsers due to `define_common_options` but I can silently forget about it if --list is present.
On `borg create`, `--verbose` does nothing interesting, so this might be what we want.